examples/json-access/json_access.description

```diff
@@ -0,0 +1,5 @@
+// Ballerina defines certain types as lax types, for which static typing rules are less strict.
```

Comment:
Ballerina defines certain types as lax types for which static typing rules are not being applied strictly.

Reply:
Not sure why we need this change? Isn't it the same?

Reply:
Chunked it because the sentence is too long